streams/test-utils/src/test/java/org/apache/kafka/streams/MockProcessorContextTest.java

```diff
@@ -231,14 +230,13 @@ public void process(final String key, final Long value) {
 
         assertFalse(context.committed());
     }
-    @SuppressWarnings({"deprecation", "unchecked"}) // TODO deprecation will be fixed in KAFKA-10437
```

Comment:
Don't need to suppress deprecation anymore, since it's moved to the class level, and don't need unchecked anymore thanks to the return-type change of `getStateStore`.